### ISSUE TITLE
Create instance form has accessibility issues.

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#373]
In order for users to successfully use the create instance for on any type of device, it should be section 508 compliant.

Acceptance criteria:
- [ ] Dropdown options should be grouped with `optgroup`.
- [ ] Fieldset should contain a legend element.
- [ ] Field set element must have an accessible name.
- [ ] Form should have some sort of submit element.

**Comment by msecret** [comment]
>
If this selection list contains groups of related options, they should be grouped with optgroup.

**Comment by msecret** [comment]
>
Fieldset does not contain a legend element. All fieldsets should contain a legend element that describes a description of the field group.

**Comment by msecret** [comment]
>
This fieldset element does not have a name available to an accessibility API. Valid names are: legend element.

**Comment by msecret** [comment]
>
This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.

**Comment by jeremiak** [comment]
A few questions:

> If this selection list contains groups of related options, they should be grouped with optgroup.

Why does this selection list contain multiple groups of related options? Since a marketplace is underneath an organization, shouldn't just the spaces for the current org show up? Currently the behavior is to show all spaces in the `SpaceStore`, but I think that's actually a bug.

> This fieldset element does not have a name available to an accessibility API. Valid names are: legend element.

How did you generate this error? I don't see anything like it in my `HTML_CodeSniffer` output.

**Comment by msecret** [comment]
I generated it with code sniffer

**Comment by msecret** [comment]
In terms of the grouping, I think you're correct, it doesn't need to be grouped. Myabe we should fix the bug where it uses all spaces in spacestore?

**Comment by jeremiak** [comment]
Agreed @msecret. I've got it fixed locally so I'll include that in this PR.

**Comment by msecret** [comment]
Checked in HTML Codesniffer and errors are all gone.
